### ISSUE TITLE
Identifier not parsed with unicode symbols [BUG]

### ISSUE THREAD

**Issue by UliPlabst** [microsoft/powerquery-parser#298]
I just god a bug report from a user of powerqueryformatter.com . He appears to be using chinese unicode symbols as identifiers and the parser throws when encountering the symbol.  

**Expected behavior**
The query he is trying to format is:  
```let 零 = 1 in 零```
The [spec](https://docs.microsoft.com/en-us/powerquery-m/m-spec-consolidated-grammar#identifiers) says an **identifier** consists of **letter_character** which itself is defined as
```
letter-character:
      A Unicode character of classes Lu, Ll, Lt, Lm, Lo, or Nl
```
I checked the symbol 零  (I don't know what it means btw) with a [unicode analyzer ](https://www.fontspace.com/unicode/char/96F6-cjk-unified-ideograph-u96f6) and it says it is of category **Lo** which should be a valid letter-character. I checked in Excel and PowerbBI and the query is parsed without problems.

**Actual behavior**
Parser throws error

**To Reproduce**
Please include the following:
* (Required) The Power Query script that triggers the issue.
*```let 零 = 1 in 零```
* (Required) Any non-default settings used in the API call(s) which trigger the issue.
none
* (Ideally) A minimal reproducible example. Can you reproduce the problem by calling a function in `src/example.ts`?

**Additional context**

**Comment by UliPlabst** [comment]
Just realized I don't even really know what kind of language that symbol is from hence the title change

**Comment by jiaopengzi** [comment]
the mail it is me.
It is very common to use Chinese in daily use. Is it possible to add the identifier in.
Really looking forward to.

**Comment by JordanBoltonMN** [comment]
I looked this over during the weekend. It seems like the site I used to scrape Lo codepoints was missing a rather large swathe of them. I've been meaning to update the regexp anyways as part of us dropping IE11 support so I bundled them into one fix.

Should be resolved as #299

**Comment by JordanBoltonMN** [comment]
Correction! I had a regression related to trailing numbers in `0.6.5`. I've just published `0.6.6` which should fixes the regression. I've published the package so once it's consumed I expect the issue to be resolved.

**Comment by UliPlabst** [comment]
Ok everything seems to be working now, I just deployed with the new parser version. Thanks for the fix!
